hack/dashboard/openshift/deploy-grafana.sh

```diff
@@ -63,25 +74,53 @@ validate_cluster() {
 		ret=1
 	}
 
-	oc get ns $MON_NS -o name || {
+	oc get ns $MON_NS -o name >/dev/null 2>&1 || {
 		fail "$MON_NS namespace missing. Is this an OpenShift cluster?"
-		info "cluster:  $(oc whoami --show-server)"
+		info "cluster:  $(oc whoami --show-server)\n"
 		ret=1
 	}
 
-	oc get crds | grep kepler || {
-		fail "Missing Kepler CRD. Is Kepler Operator Installed?"
+	oc get crds keplers.kepler.system.sustainable.computing.io || {
+		fail "Missing Kepler CRD. Is Kepler Operator Installed?\n"
 		ret=1
 	}
 
-	[[ $(oc get kepler kepler -o=jsonpath='{.status.conditions[?(@.type=="Available")].status}') == "True" ]] || {
-		fail "Mising kepler deployment. Did you create kepler instance?"
+	oc get kepler kepler >/dev/null 2>&1 || {
+		fail "Missing kepler instance. Did you forget to create kepler?"
+		info_run "oc apply -f ./config/samples/kepler.system_v1alpha1_kepler.yaml"
```

Comment:
why do we need to info this? Our sample yaml might keep on updating as and when new features are added

Reply:
Its only printing an information -right?

Reply:
agree but it might contain some configs that are experimental and not supported by the current operator version which is deployed. we can simply print that `create kepler instance from openshift console or follow our doc's`